npm/webpack-dev-server/src/loader.ts

```diff
@@ -17,7 +17,7 @@ const makeImport = (file: Cypress.Cypress['spec'], filename: string, chunkName:
   const magicComments = chunkName ? `/* webpackChunkName: "${chunkName}" */` : ''
 
   return `"${filename}": {
-    shouldLoad: () => document.location.pathname.includes("${file.absolute}"),
+    shouldLoad: () => document.location.pathname.includes("${encodeURI(file.absolute)}"),
```

Comment:
So the problem was that we weren't validating the URL correctly, but we were properly pushing the encoded URL?

Reply:
that's right!